### DISCUSSION TITLE
Remove decimal points from price

### DISCUSSION THREAD

**Discussion by amirseyedian** [bagisto/bagisto#10306] [Q&A] [answered]
### Describe the solution you'd like
core.php
```
public function currency($amount = 0)
{
    if (is_null($amount)) {
        $amount = 0;
    }

    return $this->formatPrice($this->convertPrice($amount));
}

public function formatPrice(?float $price, ?string $currencyCode = null): string
{
    if (is_null($price)) {
        $price = 0;
    }

    $currency = $currencyCode
        ? $this->getAllCurrencies()->where('code', $currencyCode)->first()
        : $this->getCurrentCurrency();

    return $this->formatCurrency($price);
}

public function formatCurrency(float $price): string
{
    $locale = app()->getLocale();
    $formatter = new \NumberFormatter($locale, \NumberFormatter::DECIMAL);
    $formatter->setAttribute(\NumberFormatter::FRACTION_DIGITS, 0); // No decimal points

    return $formatter->format($price);
}
```

**Comment by devansh-webkul** [comment, marked as answer]
We actually already have this set up for the custom formatter. 😊 You can find it here: [CurrencyFormatter.php](https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Core/src/Concerns/CurrencyFormatter.php#L55).

And as for the default formatting, it’s all handled automatically based on the locale, so there’s no extra work needed. Hope this helps, but please let me know if you need more details—I’m happy to assist!

**Reply by amirseyedian** [reply]
Thank you for your input however it would be nice to have a toggle in admin>currency section
